Userland/Libraries/LibWeb/Painting/PaintableBox.cpp

```diff
@@ -645,7 +645,7 @@ static void paint_text_fragment(PaintContext& context, Layout::TextNode const& t
         auto text = text_node.text_for_rendering();
 
         DevicePixelPoint baseline_start { fragment_absolute_device_rect.x(), fragment_absolute_device_rect.y() + context.rounded_device_pixels(fragment.baseline()) };
-        Utf8View view { text.substring_view(fragment.start(), fragment.length()) };
+        Utf8View view { text.bytes_as_string_view().substring_view(fragment.start(), fragment.length()) };
```

Comment:
`text.code_points()` seems appropriate here.

Reply:
Thanks, I  missed the `substring_view` API we have in `Utf8View`